salt/modules/cmdmod.py

```diff
@@ -269,6 +269,7 @@ def _run(cmd,
          python_shell=False,
          env=None,
          clean_env=False,
+         prepend_path=None,
```

Comment:
Can you move these new options to the end of the list of kwargs? (Here and in all of the other functions you've added `prepend_path` to.)

If people are using functions from cmdmod.py with positional arguments, then this addition will break things for them and we don't want to cause regressions.

Reply:
gotcha

Reply:
it should be _before_ kwargs, I assume?

Reply:
@rallytime pls see https://github.com/saltstack/salt/pull/44300/commits/6bd8aea8cfa5722ef78c0a16cd4ab95bccf25d98. Does this work for you?

Reply:
Yes, you've got it! :) It should be before `**kwargs`.